modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView/VersionActions.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+const VersionActions = ({actions}: IProps) => {
+	const handleActionClick = ({
+		previewURL: _previewURL,
+		title: _title,
+		useModal: _useModal,
+	}: {
+		previewURL: string;
+		title: string;
+		useModal: boolean;
+	}): void => {};
+
+	return (
+		<ClayDropDown
+			className="align-self-start pt-2"
+			closeOnClick
+			data-tooltip-align="left"
+			title={Liferay.Language.get('actions')}
+			trigger={
+				<ClayButtonWithIcon
+					borderless
+					displayType="secondary"
+					symbol="ellipsis-v"
+				/>
+			}
+		>
+			<ClayDropDown.ItemList>
+				{actions.map(
+					({
+						action,
+						actionLabel,
+						icon,
+						previewURL,
+						title,
+						useModal,
+					}) => (
+						<ClayDropDown.Item
+							key={action}
+							onClick={() =>
+								handleActionClick({previewURL, title, useModal})
+							}
+						>
+							<ClayIcon symbol={icon || ''} />
+
+							<span className="pl-3">{actionLabel}</span>
+						</ClayDropDown.Item>
+					)
+				)}
+			</ClayDropDown.ItemList>
+		</ClayDropDown>
+	);
+};
+
+interface IProps {
+	actions: IAction[];
+}
+
+interface IAction {
+	action: string;
+	actionLabel: string;
+	icon?: string;
+	previewURL: string;
```

Comment:
Hi @beltranrengifo , I prefer to use a more generic name for the url not including the name of the action. Something like actionURL could be better. What do you think?

Reply:
You're totally right @claraizquierdo 
I think I'll also remove the `useModal` until it's needed, which maybe will never occur...
What do you think?

Reply:
I agree with you @beltranrengifo . Remove it until it is needed.

Reply:
> I agree with you @beltranrengifo . Remove it until it is needed.

Perfect! So do you think we can ship this part?